documentation/navigation/explicit-navigation.md

```diff
@@ -0,0 +1,202 @@
+# Explicit navigation
+
+[In this section](https://bumble-tech.github.io/appyx/navigation/composable-navigation/#navigation-in-the-tree) we covered how changing `NavModel` will result in manipulating

```

Comment:
Let's use relative url to markdown file here too (not sure about the anchor though)